### ISSUE TITLE
Set the time and money amount in the test environment

### ISSUE THREAD

**Issue by usd-yamazaki** [paritytech/ink#126]
Please add the following functions to set the values ​​that can be obtained with `env::now()` and `env::value_transferred()` when testing off-chain?

https://github.com/paritytech/ink/blob/master/core/src/env/test.rs
```
pub fn set_now(timestamp: Moment)
pub fn set_value_transferred(value: Balance)
```

**Comment by Robbepop** [comment]
We will soon merge https://github.com/paritytech/ink/pull/108 and I think this would be a great addition afterwards.

**Comment by satyamakgec** [comment]
@usd-yamazaki It is fixed by this PR #145.
